src/lib/renderers/customFunctions.js

```diff
@@ -1,4 +1,7 @@
 export const customFunctions = {
     "Math": Math,
-    "Number": Number
+    "Number": Number,
+    "log": function () {
+        console.log('you are logging from an AG grid function');
```

Comment:
```suggestion
        console.log('you are logging from an AG grid function', ...arguments);
```
So if the user wants to call `log(1,2,3)` or whatever, they can do that, and they'll see the values directly, without setting a breakpoint.